tests/everflow/conftest.py

```diff
@@ -0,0 +1,35 @@
+import pytest
+import logging
+
+
+@pytest.fixture(autouse=True, scope="module")
+def setup_recycle_port(duthosts, tbinfo):
+    """Setup recycle port ip address on t2 topo"""
+    rec_intf = {}
+    if "t2" in tbinfo['topo']['name']:
+        for duthost in duthosts.frontend_nodes:
+            rec_intf[duthost.hostname] = {}
+            for asic in duthost.asics:
+                output = duthost.command("show ip interfaces -n {} -d all".format(asic.namespace))['stdout_lines']
```

Comment:
not all linecards are multi-asics, need to handle separately for masic and single-asic.
would be good to move this to multi_asic.py

Reply:
Fixed.  Is setup_recycle_port used anywhere else? if not and it is used for everflow, it is better to keep it where it is.

Reply:
I mean the duthost execution based on single/multi-asic condition is normally put under multi_asic.py:
output = duthost.command("show ip interfaces -n {} -d all".format(asic.namespace))['stdout_lines']